draft-ietf-core-resource-directory.md

```diff
@@ -1493,9 +1490,9 @@ Req: GET /rd-lookup/ep?et=power-node
 
 Res: 2.05 Content
 </rd/1234>;con="coap://[2001:db8:3::127]:61616";ep="node5";
-et="power-node";rt="core.ocf.power";ct="40";lt="600",
```

Comment:
what is core.ocf.power?  Where did that come from?

Reply:
It came from @petervanderstok's examples.

Reply:
If we want to use an example with "OCF" in it, we should make sure we use the registered rt values right...  I find `oic.d.light`, but no power, just `oic.d.switch`.